com.corifeus.onenote.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
```

Comment:
It should be submitted upstream.

Reply:
Why should it be upstream? I dont think it makes much of a difference, as I'd have to patch it if it were upstream (Exec=/app/bin/run.sh %u)

Reply:
Because it belongs there, it's not specific to Flathub at all.